apps/welcome/src/components/probe-page/probe-page.tsx

```diff
@@ -100,7 +100,7 @@ const ProbePage: React.FC<{
           <pre>{probe?.endpoint_id}</pre>
         </div>
         {probe?.updated !== "" && (
-          <pre>Last detected on: {probe?.updated} </pre>
+          <pre>Last detected on: {probe?.updated.substring(0, 10)} </pre>
```

Comment:
best not to parse date strings yourself, has too many edge cases. you can use parse and format from `date-fns`

```
 import { format, parse }  from "date-fns"
```
```suggestion
          <pre>Last detected on: {probe?.updated && format(parse(probe.updated), "yyyy-MM-dd")} </pre>
```